pages/user guides/financial_impact.md

```diff
@@ -0,0 +1,88 @@
+---
+toc: false
+title: Financial Impact
+keywords: financial, analysis, impact, costs, cloud, visualize, analyze, business, case
+last_updated: September, 2024
+summary: "Build your business case by visualizing the financial impact of your migration"
+sidebar: main_sidebar
+folder: userguides
+permalink: financial-impact.html
+---
+
+Tidal Accelerator empowers you to construct a compelling business case by visualizing the financial impact of your project. You can find your financial impact report by navigating to a project and clicking the 'Financial Impact' tab.
+
+## Financial Impact Overview
+
+Your Tidal financial impact report is scoped to a project, which is a collection of applications. If you're not familiar with projects check out our [documentation](https://guides.tidal.cloud/assessment-projects.html), which will walk you through setting them up and organizing your application assessments.
+
+Your financial impact report essentially has two parts. The first is the project-wide overview which includes summaries and visuals for your entire project. 
+
+{% include image.html file="financial-impact-1.png" %}
+
+The second is a drill-down section for each application in this project. 
+
+{% include image.html file="financial-impact-2.png" %}
+
+In addition, you can view an application's financial impact (the drill-down section) directly in that app's native view.
+
+{% include image.html file="financial-impact-3.png" %}
+
+For all of these visuals, the majority of the data we're summarizing is sourced from the apps themselves, so that's where you'll want to look in order to add and tweak data. The only current exception to this is cloud instance targets, which are included in the cloud costs of your app. For more information about generating cloud instance targets, see our [guide](https://guides.tidal.cloud/instance-recommendation.html).
```

Comment:
This isn't very clear. It might be simpler to say that we derive the financial analysis based on information gathered from the application itself and the server via cloud instance targets. Then, link to the guide on the term cloud instance target; I don't think we always need to call out the fact that it is a guide. Instead, we can treat it as a "find out more button."